src/v/cluster/topics_frontend.cc

```diff
@@ -480,6 +480,8 @@ ss::future<topic_result> topics_frontend::replicate_create_topic(
 ss::future<std::vector<topic_result>> topics_frontend::delete_topics(
   std::vector<model::topic_namespace> topics,
   model::timeout_clock::time_point timeout) {
+    vlog(clusterlog.info, "Delete topics {}", topics);
```

Comment:
looks fine, but doesn't match the title of the PR so just double checking it is intentional

Reply:
Yep, this one went in because with the other messages turned down to debug, there ended up being nothing in the log to indicate when a topic was deleted.

Reply:
makes sense. i should have realized that